README.md

```diff
@@ -46,6 +46,12 @@ kamel install
 
 This will configure the cluster with the Camel K custom resource definitions and install the operator on the current namespace.
 
+If requires to uninstall Camel K from the OpenShift or Kubernetes, it's nessesary to run following command using "oc" or "kubectl" tool:
```

Comment:
Maybe better moving this to the bottom, in a specific paragraph about uninstallation

Reply:
Moved it.